### ISSUE TITLE
undefined method `convert_inline_break' for #<Asciidoctor::FB2::Converter:0x00005628fef4df30>

### ISSUE THREAD

**Issue by dismine** [slonopotamus/asciidoctor-fb2#14]
```
undefined method `convert_inline_break' for #<Asciidoctor::FB2::Converter:0x00005628fef4df30>
Did you mean?  convert_inline_kbd
               convert_inline_menu
               convert_inline_image
               convert_inline_button
  Use --trace for backtrace
```

```
asciidoctor-fb2 -v
Asciidoctor FB2 0.2.1 using Asciidoctor 2.0.10 [https://asciidoctor.org]
Runtime Environment (ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-linux-gnu]) (lc:UTF-8 fs:UTF-8 in:UTF-8 ex:UTF-8)
```

**Comment by slonopotamus** [comment]
Oh c'moon. I possibly should just log a warning for now for unsupported elements =/

**Comment by slonopotamus** [comment]
FB2 doesn't have anything that could emulate inline break. So, I'm going to make them a no-op.

**Comment by slonopotamus** [comment]
Fixed in 0.2.2 together with verses and quotes.

**Comment by dismine** [comment]
Thank you for your hard work!
